### PR TITLE
Add subtraction finalization endpoint

### PR DESCRIPTION
**Pull request by tianshengsui**
[ch424]

### CODE REVIEW

**Review by igboyes — CHANGES_REQUESTED** [review]
Thanks. One more thing.

**Comment by tianshengsui** [comment]
@igboyes You didn't select any code.

File: virtool/subtractions/api.py [hunk 1]
```diff
@@ -303,3 +303,29 @@ async def remove(req):
...
+
+    if document is None:
+        return not_found()
+
+    updated_document = await db.subtraction.find_one_and_update({"_id": subtraction_id}, {
+        "$set": {
+            "gc": data["gc"],
+            "ready": True
+        }
+    })
```
Comment (igboyes) [inline virtool/subtractions/api.py:329]: Return `409` if subtraction has already been finalized. Message: _Subtraction has already been finalized_.

**Review by igboyes — CHANGES_REQUESTED** [review]

**Comment by igboyes** [comment]
> @igboyes You didn't select any code.

I think I typed my comment, but didn't save. Added review.

**Review by igboyes — APPROVED** [review]

**Pull request merged**